src/app/libp2p_helper/LICENSE

```diff
@@ -1,201 +0,0 @@
-                                 Apache License
```

Comment:
can you un-remove the license?

Reply:
@bkase my intuition was that LICENSE at the root of Mina repository applies here unless specified otherwise.

Reply:
https://github.com/MinaProtocol/mina/blob/compatible/LICENSE this file

Reply:
sounds good 👍